local/rest_api_gcbm/gcbm.py

```diff
@@ -174,7 +187,14 @@ class GCBMSimulation:
     def __init__(self):
         # create a global index
         self.dirpath = "input/test-run"
+        self.config_dir_path = "templates"
         self.files = {}
+        
+        # Tracks the output path for the input received through `add_file` method
+        self.json_paths = {}
+
+        # TODO: Once categories are changed from strings to Enums, we should find a better way to have supported categories
+        self.supported_categories = ["classifiers", "disturbances", "miscellaneous"]
```

Comment:
We need to support input databases also. If you rename `/tests/tiffs/new_demo_run` $\rightarrow$ `tests/reference/new_demo_run` then it makes sense to include files in addition to `tiffs`.

Reply:
PS. thank you for change this to lowercase - CamelCase filenames drive me nuts.

Reply:
P.P.S I also got sick of the unnecessary `_moja` suffix and removed it from the `linux-demo` reference simulation on #165

Reply:
> We need to support input databases also.

Got it! I understand that it's a TODO, and I'll add this feature in the next commit. Thanks for letting me know about the `_moja` suffix removal, since it's merged now - I'll rebase and fix these suffixes.

Reply:
Fixed! :)